loops.js

```diff
@@ -33,6 +38,13 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
+for (let i = 0; i < 16; i++) {
+    if (i % 2 === 0) {
+        console.log(i + " is even")
+    } else if (i % 2 !== 0) {
```

Comment:
Ik zie in de uitwerkingen een vergelijkbare oplossing met de tweede controle op 'oneven'. Zelf had ik die achterwege gelaten, omdat ik ervan uitgegaan ben dat als het niet even is, dat het oneven is.